dnf5/commands/list/list.cpp

```diff
@@ -145,8 +145,8 @@ void ListCommand::run() {
     auto & ctx = get_context();
     auto & config = ctx.base.get_config();
 
-    libdnf5::rpm::PackageQuery full_package_query(ctx.base);
-    libdnf5::rpm::PackageQuery base_query(ctx.base);
+    libdnf5::rpm::PackageQuery full_package_query(ctx.base, libdnf5::sack::ExcludeFlags::IGNORE_VERSIONLOCK);
```

Comment:
Probably needed also for advisory list, info? And what about plugins like changelog, needs-restarting? Or maybe, if I understand it correctly, we should make sure all existing package queries related to non-transactional operations have the flag present?

Reply:
Well not all - for example `check-upgrade` does not create a transaction but we still want to keep the results in sync with what `dnf upgrade` would do.
But generally you are right - I'll add the flag to missing places.